h/static/scripts/streamsearch.coffee

```diff
@@ -30,8 +27,6 @@ class StreamSearchController
 
     $scope.shouldShowThread = (container) -> true
 
-    streamer.send({filter: streamfilter.getFilter()})
```

Comment:
I think you still want to send this, but just change it to use past_data: none. No?